<!DOCTYPE html>
<html lang="en-us">
   <head>
      <meta name="viewport" content="width=device-width,initial-scale=1,viewport-fit=cover">
      <title>Research - Liu Lab</title>
      <meta name="description" content=".frontmatter { text-align: justify; text-justify: inter-word; } img { padding-left: 3rem; } .section { padding-bottom: 0rem; }">
      <link rel="icon" target="_blank" href="./image/favicon.png">
      <link rel="canonical" target="_blank" href="https://hbLiulab.org/">
      <meta property="og:title" content="">
      <meta property="og:description" content="Lab website for the <b>Liu</b> Lab">
      <meta property="og:type" content="website">
      <meta property="og:url" content="/">
      <link rel="stylesheet" target="_blank" href="./css/bulma.min.css">
      <link rel="stylesheet" target="_blank" href="./css/style.css">
      <link rel="stylesheet" target="_blank" href="./css/all.css">
      <link rel="preconnect" target="_blank" href="https://fonts.gstatic.com/">
      <link target="_blank" href="./css/css2" rel="stylesheet">
      <link rel="stylesheet" target="_blank" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
      <script src="./css/jquery.min.js"></script>
   </head>
   <body>
      <!-- Header -->
      <header>
         <div class="container">
            <nav class="navbar has-shadow" role="navigation">
               <div class="navbar-brand">
                  <a target="_blank" href="https://hbLiulab.org" title="home" class="navbar-item">
                     <span class="logo">
                        <h1><b>Liu</b> Lab</h1>
                        <h3>University of Rochester</h3>
                     </span>
                  </a>
                  <a role="button" class="navbar-burger" aria-label="menu" aria-expanded="false"><span aria-hidden="true"></span><span aria-hidden="true"></span><span aria-hidden="true"></span></a>
               </div>
               <div class="navbar-menu">
                  <div class="navbar-start">
                     <a href="https://hbLiulab.org" class="navbar-item">Home</a>
                     <a href="https://hbLiulab.org/team.html" class="navbar-item">Team</a>
                     <a href="https://hbLiulab.org/research.html" class="navbar-item">Research</a>
                     <a href="https://hbLiulab.org/publications.html" class="navbar-item">Publications</a>
                     <a href="https://hbLiulab.org/resources.html" class="navbar-item">Resources</a>
                     <a href="https://hbLiulab.org/positions.html" class="navbar-item">Positions</a>
                     <a href="https://hbLiulab.org/contact.html" class="navbar-item">Contact</a>
<!--                 <a href="https://hbliu.github.io" target="_blank" class="navbar-item">People</a> -->
<!--                 <a href="https://github.com/hbliu?tab=repositories" target="_blank" class="navbar-item">Software</a> -->
                  </div>
               </div>
            </nav>
         </div>
         <script>$(document).ready(function(){$(".navbar-burger").click(function(){$(".navbar-burger").toggleClass("is-active");$(".navbar-menu").toggleClass("is-active");})})</script>
      </header>
      <!-- Main content -->
      <main id="content">
         <style>.frontmatter{text-align:justify;text-justify:inter-word}img{padding-left:3rem}.section{padding-bottom:0}</style>
         <section class="section">
            <div class="container">
               <div class="content">
                  <center>
                     <h1 id="RegVar">Dissecting regulatory non-coding variants in human disease</h1>
                  </center>
                  <div class="columns">
                     <div class="column">
                        <video src="./image/pub/2025.Liu.Science.mp4" loop="true" autoplay="autoplay" controls="controls" style="max-width: 730px;" muted="" poster=""> </video>
                     </div>
                     <div class="column">
                        <br>
                        <p class="frontmatter">Genome-wide association studies (GWAS) have identified numerous DNA sequence variants associated with complex human diseases. However, over 90% of disease-associated variants reside in noncoding genome regions, and their functions in complex diseases remain largely unknown—a problem often referred to as the ‘variant-to-function’ problem. To dissect the regulatory function of noncoding DNA sequence variants, we perform large-scale genome-wide association studies (GWAS) and quantitative trait loci (QTL) studies for gene expression, DNA methylation and DNA accessibility. We discovered that epigenome (DNA methylation) explains a larger fraction of heritability than gene expression. Further, we found the allele-specific expression and open chromatin driven by GWAS variants associated with kidney disease. To further identify disease-causal genes, I proposed a <a target="_blank" href="https://susztaklab.com/GWAS2M/">Disease Genetic Scorecard</a> and prioritized >600 kidney disease genes targeted by both coding variants and regulatory variants, including <em>SLC47A1</em>, whose causal role was defined in knockout mice model and human individuals carrying loss-of-function variants. (<b>Liu</b> et al., 2025 <a target="_blank" href="https://www.science.org/doi/full/10.1126/science.adp4753">Science</a>; <b>Liu</b> et al., 2022 <a target="_blank" href="https://www.nature.com/articles/s41588-022-01097-w">Nature Genetics</a>; <a target="_blank" href="https://susztaklab.com/GWAS2M/">Project page</a>).</p>
                     </div>
                  </div>
               </div>
               <nav class="navbar has-shadow" role="navigation"></nav>
            </div>
         </section>
          <section class="section">
            <div class="container">
               <div class="content">
                  <center>
                     <h1 id="EpiReg">Epigenetic regulation in aging and age-related disease</h1>
                  </center>
                  <div class="columns">
                      <div class="column">
                        <br>
                        <p class="frontmatter">Epigenetic regulators play central roles in aging and age-related diseases. By applying our bioinformatic tools to large-scale epigenome data, we explore the epigenetic changes in aging and age-related chronic diseases. In previous studies, we have identified disease-critical epigenetic regulatory elements, including DNA methylation, histone modifications, super-enhancers, and long non-coding RNAs, and explored their roles in complex human diseases (e.g. kidney disease, diabetes, and cancer) (Liang, <b>Liu</b> et al. 2024 <a target="_blank" href="https://www.nature.com/articles/s41467-024-53798-x">Nature Communications</a>; Yan, <b>Liu</b> et al. 2024 <a target="_blank" href="https://www.nature.com/articles/s41467-024-45295-y">Nature Communications</a>; Xu, <b>Liu</b> et al. 2019 <a target="_blank" href="https://www.nature.com/articles/s41419-019-2137-5">Cell Death &amp; Disease</a>; Xiong,…, <b>Liu</b> et al. 2017 <a target="_blank" href="https://academic.oup.com/nar/article/45/D1/D888/2605746">NAR</a>; Wei,…, <b>Liu</b> et al. 2016 <a target="_blank" href="https://academic.oup.com/nar/article/44/D1/D172/2503054">NAR</a>; Lv, <b>Liu</b> et al. 2013 <a target="_blank" href="https://academic.oup.com/nar/article/41/22/10044/2438380">NAR</a>; Lv, <b>Liu</b> et al. 2012 <a target="_blank" href="https://academic.oup.com/nar/article/40/D1/D1030/2903287">NAR</a>; Zhang, <b>Liu</b> et al. 2011 <a target="_blank" href="https://academic.oup.com/nar/article/39/9/e58/1254752">NAR</a>; Zhang, Lv, <b>Liu</b> et al. 2010 <a target="_blank" href="https://academic.oup.com/nar/article/38/suppl_1/D149/3112313">NAR</a>).</p>
                     </div>
                    <div class="column">
                        <a target="_blank" href="https://www.nature.com/articles/s41588-022-01097-w" target="_blank"><img src="./image/research/Epigenetic.Regulation.png"></a>
                     </div>
                  </div>
               </div>
               <nav class="navbar has-shadow" role="navigation"></nav>
            </div>
         </section>
         <section class="section">
            <div class="container">
               <div class="content">
                  <center>
                     <h1 id="CelOri">Identifying cellular origins of complex human disease</h1>
                  </center>
                  <div class="columns">
                     <div class="column">
                        <a target="_blank" href="https://www.science.org/doi/full/10.1126/science.adp4753" target="_blank"><img src="./image/research/Cellular.Origins.png"></a>
                     </div>
                     <div class="column">
                        <p class="frontmatter">The causal cell type and regulatory mechanisms are poorly understood for complex diseases. To identify causal cell types, we developed a series of bioinformatic software, such as <a target="_blank" href="https://github.com/hbliu/Open4Gene">Open4Gene</a> for linking enhancers to target genes from single cell multiome data (<b>Liu</b> et al., 2025 <a target="_blank" href="https://www.science.org/doi/full/10.1126/science.adp4753">Science</a>); <a target="_blank" href="http://fame.edbc.org/smart/">SMART</a> for de novo identifying tissue/cell-specific methylated regions from whole genome bisulfite sequencing data (<b>Liu</b> et al. 2016 <a target="_blank" href="https://academic.oup.com/nar/article/44/1/75/2499653">NAR</a>); QDMR for identifying differentially methylated regions from array-based methylation data (Zhang, <b>Liu</b> et al. 2011 <a target="_blank" href="https://academic.oup.com/nar/article/39/9/e58/1254752">NAR</a>; <a target="_blank" href="http://fame.edbc.org/qdmr/">Project page</a>). Further, we generated single-nucleus transposase-accessible chromatin with sequencing (snATAC-seq), single-cell RNA sequencing (scRNA), and spatially resolved transcriptomics. The analysis illustrated the crucial roles of kidney cells (e.g., proximal tubule cells) and cell type-specific genes (e.g., <em>SLC47A1</em>) in kidney injury and fibrosis. (<b>Liu</b> et al., 2025 <a target="_blank" href="https://www.science.org/doi/full/10.1126/science.adp4753">Science</a>; <b>Liu</b> et al., 2022 <a target="_blank" href="https://www.nature.com/articles/s41588-022-01097-w">Nature Genetics</a>; Sheng,…,<b>Liu</b> et al., 2021 <a target="_blank" href="https://www.nature.com/articles/s41588-021-00909-9">Nature Genetics</a>; Miao,…, <b>Liu</b> et al. 2021, <a target="_blank" href="https://www.nature.com/articles/s41467-021-22266-1">Nature Comms</a>; Doke,…, <b>Liu</b> et al., 2021 <a target="_blank" href="https://www.jci.org/articles/view/141801">JCI</a>; Dhillion,…, <b>Liu</b> et al., <a target="_blank" href="https://www.sciencedirect.com/science/article/pii/S1550413120306069?via%3Dihub">Cell Metabolism</a>; Abedini,…, <b>Liu</b> et al. 2022 <a target="_blank" href="https://www.biorxiv.org/content/10.1101/2022.10.24.513598v1">BioRxiv</a>).</p>
                     </div>
                  </div>
               </div>
               <nav class="navbar has-shadow" role="navigation"></nav>
            </div>
         </section>
         <section class="section">
            <div class="container">
               <div class="content">
                  <center>
                     <h1 id="EpiDev">Mapping epigenetic dynamics in mammalian development</h1>
                  </center>
                  <div class="columns">
                     <div class="column">
                        <br><br>
                        <p class="frontmatter">Epigenetic marking systems confer precise regulation of gene expression during cell development and differentiation. DNA methylation and histone modification undergo dynamics during mammalian development (<b>Liu</b> et al. 2014 <a target="_blank" href="https://academic.oup.com/database/article/doi/10.1093/database/bat084/2633757">Database</a>; <b>Liu</b> et al. 2013 <a target="_blank" href="https://www.nature.com/articles/srep02576">Sci. Rep.</a>). Using knockout mice of <em>Dnmt3a/b</em>, we demonstrated essential roles of DNA methylation in decommissioned fetal enhancers linking to kidney disease (Guan, <b>Liu</b> et al. 2020 <a target="_blank" href="https://jasn.asnjournals.org/content/31/4/765">JASN</a>). Using knockout mice of <em>Tet2/3</em>, we demonstrated critical roles of hydroxymethylation for nephron progenitor differentiation and nephron endowment (Liang,..., <b>Liu</b> et al. 2023 <a target="_blank" href="https://doi.org/10.1681/ASN.2022040460">JASN</a>). These studies highlighted locus-specific convergence of genetic, epigenetic, and developmental elements in disease development.</p>
                     </div>
                     <div class="column">
                        <a target="_blank" href="https://jasn.asnjournals.org/content/31/4/765" target="_blank"><img src="./image/research/JASN2020.jpg"></a>
                     </div>
                  </div>
               </div>
               <nav class="navbar has-shadow" role="navigation"></nav>
            </div>
         </section>
      </main>
      <!-- Footer information -->
      <footer>
         <div class="container">
            <nav class="navbar has-shadow" role="navigation"></nav>
            <center>
               <a target="_blank" href="https://github.com/hbLiulab" target="_blank" class="fa fa-github"></a>
               <a target="_blank" href="https://twitter.com/Hongbo919<b>Liu</b>" target="_blank" class="fa fa-twitter"></a>
            </center>
            <center>Copyright © 2025 Liu Lab at URMC. All rights reserved.</center>
         </div>
      </footer>
   </body>
</html>
