<!DOCTYPE html>
<html lang="en-us">
   <head>
      <meta name="viewport" content="width=device-width,initial-scale=1,viewport-fit=cover">
      <title>Resources - Liu Lab</title>
      <meta name="description" content=".frontmatter { text-align: justify; text-justify: inter-word; } img { padding-left: 3rem; } .section { padding-bottom: 0rem; }">
      <link rel="icon" href="./image/favicon.png">
      <link rel="canonical" href="https://hbliulab.org/">
      <meta property="og:title" content="">
      <meta property="og:description" content="Lab website for the Liu Lab">
      <meta property="og:type" content="website">
      <meta property="og:url" content="/">
      <link rel="stylesheet" href="./css/bulma.min.css">
      <link rel="stylesheet" href="./css/style.css">
      <link rel="stylesheet" href="./css/all.css">
      <link rel="preconnect" href="https://fonts.gstatic.com/">
      <link href="./css/css2" rel="stylesheet">
      <script src="./css/jquery.min.js"></script>
      <script type="text/javascript" src="https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js"></script>
      <script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
      </style>
   </head>
   <body>
      <!-- Header -->
      <header>
         <div class="container">
            <nav class="navbar has-shadow" role="navigation">
               <div class="navbar-brand">
                  <a href="https://hbliulab.org" title="home" class="navbar-item">
                     <span class="logo">
                        <h1><b>Liu</b> Lab</h1>
                        <h3>University of Rochester</h3>
                     </span>
                  </a>
                  <a role="button" class="navbar-burger" aria-label="menu" aria-expanded="false"><span aria-hidden="true"></span><span aria-hidden="true"></span><span aria-hidden="true"></span></a>
               </div>
               <div class="navbar-menu">
                  <div class="navbar-start">
                     <a href="https://hbLiulab.org" class="navbar-item">Home</a>
                     <a href="https://hbLiulab.org/team.html" class="navbar-item">Team</a>
                     <a href="https://hbLiulab.org/research.html" class="navbar-item">Research</a>
                     <a href="https://hbLiulab.org/publications.html" class="navbar-item">Publications</a>
                     <a href="https://hbLiulab.org/resources.html" class="navbar-item">Resources</a>
                     <a href="https://hbLiulab.org/positions.html" class="navbar-item">Positions</a>
                     <a href="https://hbLiulab.org/contact.html" class="navbar-item">Contact</a>
                     <!--                 <a href="https://hbliu.github.io" target="_blank" class="navbar-item">People</a> -->
                     <!--                 <a href="https://github.com/hbliu?tab=repositories" target="_blank" class="navbar-item">Software</a> -->
                  </div>
               </div>
            </nav>
         </div>
         <script>$(document).ready(function(){$(".navbar-burger").click(function(){$(".navbar-burger").toggleClass("is-active");$(".navbar-menu").toggleClass("is-active");})})</script>
      </header>
      <!-- Main content -->
      <main id="content">
         <style>.frontmatter{text-align:justify;text-justify:inter-word}img{padding-left:3rem}.section{padding-bottom:0}</style>
         <section class="section">
            <div class="container">
               <div class="content">
                  <h1 id="welcome-to-the-Liu-lab" style="text-align: center">Software</h1>
                  <h4 class="subtitle is-4"></h4>
                  <table style="width:100%">
                     <tbody>
                        <tr style="text-align: left;">
                           <td style="text-align:left; vertical-align:top; width:12%">
                           		<a target="_blank" href="https://github.com/hbliu/Open4Gene"><img alt="" src="./image/Resources/Open4Gene.png" width="200"></a>
                           </td>
                          
                           <td style="text-align:left; vertical-align:top; width:55%">
                              <p><a target="_blank" href="https://github.com/hbliu/Open4Gene">Open4Gene: Hurdle Model-based Method for Peak-to-Gene Linkage Analysis</a><br></p>
                              <p style="font-size:80%;">Open4Gene is an Hurdle model-based method for peak-to-gene linkage analysis from single-cell multiome (ATAC+RNA) datasets. Open4Gene applies two-component mixture Hurdle model to account for excess zeros in single-nucleus RNA data and model linkage between peak open chromatin (ATAC) and gene expression (RNA).<br><br>
                                 The Open4Gene source code (R package) and tutorial are available at <a target="_blank" href="https://github.com/hbliu/Open4Gene">https://github.com/hbliu/Open4Gene</a>.<br><br>
                                 Our manuscript describing Open4Gene was published as: Liu et al., <a target="_blank" href="https://www.science.org/doi/full/10.1126/science.adp4753">Kidney multiome-based genetic scorecard reveals convergent coding and regulatory variants.</a> <strong>Science</strong> (2025). PMID: <a target="_blank" href="https://pubmed.ncbi.nlm.nih.gov/39913582/">39913582</a><br>
                              </p>
                              <p></p>
                           </td>
                           <td style="text-align:left; vertical-align:top; width:33%">
                           		<a target="_blank" href="https://github.com/hbliu/Open4Gene"><img alt="" src="./image/Resources/Open4Gene.Detail.png" width="450">
                           </td>
                        </tr>

                        <tr style="text-align: left;">
                           <td style="text-align:left; vertical-align:top; width:12%">
                           		<a target="_blank" href="https://github.com/hbliu/SMART2"><img alt="" src="./image/Resources/SMART.png" width="200"></a>
                           </td>
                           
                           <td style="text-align:left; vertical-align:top; width:55%">
                              <p><a target="_blank" href="https://github.com/hbliu/SMART2">SMART: Specific Methylation Analysis and Report Tool</a><br></p>
                              <p style="font-size:80%;">SMART is a quantitative method for the identification of tissue or cell type-specific DNA methylation marks from bisulfite sequencing datasets. This tool focuses on the methylation-based genome segmentation, <i>de novo</i> identification of differentially methylated regions (DMRs), and the identification of DMRs from predefined regions of interest.<br><br>
                                 The SMART source code (Python package) and tutorial are available at <a target="_blank" href="https://github.com/hbliu/SMART2">https://github.com/hbliu/SMART2</a>.<br><br>
                                 Our manuscript describing Open4Gene was published as: Liu et al., <a target="_blank" href="https://academic.oup.com/nar/article/44/1/75/2499653">Systematic identification and annotation of human methylation marks based on bisulfite sequencing methylomes reveals distinct roles of cell type-specific hypomethylation in the regulation of cell identity genes.</a> <strong>Nucleic Acids Research</strong> (2016). PMID: <a target="_blank" href="https://pubmed.ncbi.nlm.nih.gov/26635396/">26635396</a><br>
                              </p>
                              <p></p>
                           </td>
                           <td style="text-align:left; vertical-align:top; width:33%">
                           		<a target="_blank" href="http://fame.edbc.org/smart"><img alt="" src="./image/Resources/SMART2_Workflow.jpg" width="450">
                           </td>
                        </tr>

                        <tr style="text-align: left;">
                           <td style="text-align:left; vertical-align:top; width:12%">
                           		<a target="_blank" href="https://github.com/hbliu/QDMR"><img alt="" src="./image/Resources/QDMR.png" width="200"></a>
                           </td>
                           
                           <td style="text-align:left; vertical-align:top; width:55%">
                              <p><a target="_blank" href="https://github.com/hbliu/QDMR">QDMR: Quantitative Differentially Methylated Region</a><br></p>
                              <p style="font-size:80%;">QDMR is an entropy-based method for the identification of differentially methylated regions by entropy. This tool focuses on quantifying methylation differences by entropy and identifying differentially methylated regions (DMRs) within predefined regions of interest.<br><br>
                                 The QDMR source code (Java package) and tutorial are available at <a target="_blank" href="https://github.com/hbliu/QDMR">https://github.com/hbliu/QDMR</a>.<br><br>
                                 Our manuscript describing QDMR was published as: Zhang*, Liu* et al., <a target="_blank" href="https://academic.oup.com/nar/article/39/9/e58/1254752">QDMR: a quantitative method for identification of differentially methylated regions by entropy.</a> <strong>Nucleic Acids Research</strong> (2011). PMID: <a target="_blank" href="https://pubmed.ncbi.nlm.nih.gov/21306990/">21306990</a><br>
                              </p>
                              <p></p>
                           </td>
                           
                           <td style="text-align:left; vertical-align:top; width:33%">
                           		<a target="_blank" href="http://fame.edbc.org/qdmr"><img alt="" src="./image/Resources/QDMR.Interface.png" width="450">
                           </td>
                        </tr>
                     </tbody>
                  </table>
                  
                  <br>
               </div>
               
               <div class="content">
                  <h1 id="welcome-to-the-Liu-lab" style="text-align: center">Genomic and Epigenomic Atlas</h1>
                  <h4 class="subtitle is-4"></h4>
                  <table style="width:100%">
                     <tbody>
                        <tr style="text-align: left;">
                           <td style="text-align:left; vertical-align:top; width:12%">
                           		<a target="_blank" href="https://susztaklab.com/GWAS2M/KDG.Scorecard.php"><img alt="" src="./image/Resources/ScoreCard.png" width="200"></a>
                           </td>
                          
                           <td style="text-align:left; vertical-align:top; width:55%">
                              <p><a target="_blank" href="https://susztaklab.com/GWAS2M/KDG.Scorecard.php">eGFRcrea GWAS and Genetic Scorecard Atlases</a><br></p>
                              <p style="font-size:80%;">The eGFRcrea GWAS and Genetic Scorecard Atlases provide a comprehensive mapping and prioritization of GWAS variants associated with kidney function, including functional variants, target genes, and cell type-specific regulatory circuits.<br><br>
                                 These atlases include the <a target="_blank" href="http://www.susztaklab.com/GWAS2M">eGFRcrea GWAS (sample size N = 2.2 million)</a>, <a target="_blank" href="http://www.susztaklab.com/GWAS">eGFRcrea GWAS (N = 1.5 million)</a>, <a target="_blank" href="http://www.susztaklab.com/Kidney_eQTL/eQTLmeta.php">Kidney eQTL</a>, <a target="_blank" href="http://www.susztaklab.com/Kidney_meQTL">Kidney meQTL</a>, <a target="_blank" href="http://www.susztaklab.com/Human_snATAC">Kidney Cell Open Chromatin Atlas</a>, and <a target="_blank" href="https://susztaklab.com/GWAS2M/KDG.Scorecard.php">Kidney Disease Genetic Scorecard</a>, which Hongbo constructed during postdoctoral training in the <a target="_blank" href="https://www.med.upenn.edu/susztaklab/">Susztak Lab</a> at the University of Pennsylvania.<br><br>
                                 Our manuscripts describing eGFRcrea GWAS and Kidney Disease Genetic Scorecard were published as:<br>
                                 Liu et al., <a target="_blank" href="https://www.science.org/doi/full/10.1126/science.adp4753">Kidney multiome-based genetic scorecard reveals convergent coding and regulatory variants.</a> <strong>Science</strong> (2025). PMID: <a target="_blank" href="https://pubmed.ncbi.nlm.nih.gov/39913582/">39913582</a><br>
                                 Liu et al., <a target="_blank" href="https://www.science.org/doi/full/10.1126/science.adp4753">Epigenomic and transcriptomic analyses define core cell types, genes and targetable mechanisms for kidney disease.</a> <strong>Nature Genetics</strong> (2022). PMID: <a target="_blank" href="https://pubmed.ncbi.nlm.nih.gov/35710981/">35710981</a>
                                 <br>
                              </p>
                              <p></p>
                           </td>
                           <td style="text-align:left; vertical-align:top; width:33%">
                           		<a target="_blank" href="https://susztaklab.com/GWAS2M/KDG.Scorecard.php"><img alt="" src="./image/Resources/ScoreCard.Detail.png" width="450">
                           </td>
                        </tr>

                        <tr style="text-align: left;">
                           <td style="text-align:left; vertical-align:top; width:12%">
                           		<a target="_blank" href="http://fame.edbc.org/methymark"><img alt="" src="./image/Resources/MethyMark.png" width="200"></a>
                           </td>
                          
                           <td style="text-align:left; vertical-align:top; width:55%">
                              <p><a target="_blank" href="http://fame.edbc.org/methymark">Human Methylation Mark Atlas across Tissues and Cell Types</a><br></p>
                              <p style="font-size:80%;">The Human Methylation Mark Atlas provides the DNA methylation marks identified across 50 human tissues and cell types by applying <a target="_blank" href="https://github.com/hbliu/SMART2">SMART</a> to large-scale bisulfite sequencing datasets.<br><br>
                                 The Human Methylation Mark Atlas is available at <a target="_blank" href="http://fame.edbc.org/methymark">http://fame.edbc.org/methymark</a>.<br><br>
                                 Our manuscript describing this atlas was published as: Liu et al., <a target="_blank" href="https://academic.oup.com/nar/article/44/1/75/2499653">Systematic identification and annotation of human methylation marks based on bisulfite sequencing methylomes reveals distinct roles of cell type-specific hypomethylation in the regulation of cell identity genes.</a> <strong>Nucleic Acids Research</strong> (2016). PMID: <a target="_blank" href="https://pubmed.ncbi.nlm.nih.gov/26635396/">26635396</a><br>
                              </p>
                              <p></p>
                           </td>
                           <td style="text-align:left; vertical-align:top; width:33%">
                           		<a target="_blank" href="http://fame.edbc.org/methymark"><img alt="" src="./image/Resources/MethyMark.Summary.png" width="450">
                           </td>
                        </tr>

                        <tr style="text-align: left;">
                           <td style="text-align:left; vertical-align:top; width:12%">
                           		<a target="_blank" href="http://db.edbc.org/DevMouse"><img alt="" src="./image/Resources/Devmouse.png" width="200"></a>
                           </td>
                           
                           <td style="text-align:left; vertical-align:top; width:55%">
                              <p><a target="_blank" href="https://github.com/hbliu/QDMR">DevMouse: Mouse Developmental Methylome Database</a><br></p>
                              <p style="font-size:80%;">DevMouse is the mouse developmental methylome database, which provides DNA methylomes in temporal order and quantitative analysis of methylation dynamics during mouse development.<br><br>
                                 The DevMouse database is available at <a target="_blank" href="http://db.edbc.org/DevMouse">http://db.edbc.org/DevMouse</a>.<br><br>
                                 Our manuscript describing DevMouse was published as: Liu et al., <a target="_blank" href="https://academic.oup.com/database/article/doi/10.1093/database/bat084/2633757">DevMouse, the mouse developmental methylome database and analysis tools.</a> <strong>Database</strong> (2014). PMID: <a target="_blank" href="https://pubmed.ncbi.nlm.nih.gov/24408217/">24408217</a><br>
                              </p>
                              <p></p>
                           </td>
                           
                           <td style="text-align:left; vertical-align:top; width:33%">
                           		<a target="_blank" href="http://db.edbc.org/DevMouse"><img alt="" src="./image/Resources/Devmouse.Summary.png" width="450">
                           </td>
                        </tr>
                     </tbody>
                  </table>
                  
                  <br>
               </div>

            </div>
         </section>
      </main>
      <!-- Footer information -->
      <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
      <footer>
         <div class="container">
            <nav class="navbar has-shadow" role="navigation"></nav>
            <center>
               <a href="https://github.com/hbliulab" target="_blank" class="fa fa-github"></a>
               <a href="https://twitter.com/Hongbo919Liu" target="_blank" class="fa fa-twitter"></a>
            </center>
            <center>Copyright Â© 2026 Liu Lab at URMC. All rights reserved.</center>
         </div>
      </footer>
   </body>
</html>
